tests/behat.yml.dist

```diff
@@ -48,7 +48,7 @@ default:
     EC\Behat\PoetryExtension:
       application:
         base_url: '{{ behat.base_url }}'
-        endpoint: '/dgt_ftt_translator/dgt_service_callback'
+        endpoint: '/tmgmt_dgt_connector/server'
```

Comment:
We need to keep and eye on this change and revert it (or change it) when we finish https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-1601